### PR TITLE
fix(OpenAPI): RHICOMPL-3322 fix response example

### PR DESCRIPTION
**Pull request by romanblanco**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: spec/swagger_helper.rb [hunk 1]
```diff
@@ -22,10 +22,17 @@
...
 
-def autogenerate_examples(example)
-  example.metadata[:response][:examples] = {
-    'application/vnd.api+json' => JSON.parse(response.body,
-                                             symbolize_names: true)
+def autogenerate_examples(example, name='Response example', summary='', description='')
+  example.metadata[:response][:content] = {
+    'application/json': {
+      examples: {
+        "#{name}": {
```
Comment (vkrizan) [inline spec/swagger_helper.rb:29]: The name could be a bit verbose.

File: spec/swagger_helper.rb [hunk 2]
```diff
@@ -22,10 +22,17 @@
   }
 end
 
-def autogenerate_examples(example)
-  example.metadata[:response][:examples] = {
-    'application/vnd.api+json' => JSON.parse(response.body,
-                                             symbolize_names: true)
+def autogenerate_examples(example, name='Response example', summary='', description='')
```
Comment (vkrizan) [inline spec/swagger_helper.rb:25]:
```
spec/swagger_helper.rb:25:1: C: Metrics/MethodLength: Method has too many lines. [11/10]
```

**Review by vkrizan — APPROVED** [review]
Besides the lint and one suggestion LGTM.

File: spec/swagger_helper.rb (hunk 1 above)
Comment (romanblanco) [reply spec/swagger_helper.rb]: Renamed to `label`.

**Pull request merged**